src/body/BodyDom.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import {Body} from "../models";
+
+export default class BodyDom extends React.PureComponent<Body> {
+    render(): React.ReactElement<React.JSXElementConstructor<Body>> {
```

Comment:
I'd say that `React.ReactElement<React.JSXElementConstructor<Body>>` is "overtyped" let's just write `React.ReactNode`.

Reply:
updated